tasks/gitlab_permissions.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: "Set owner of gitlab directory {{ gitlab_install_path }}"
+  file:
+    path: "{{ gitlab_install_path | mandatory }}"
+    owner: git
+    recurse: yes
+
+- name: Place gitlab config
+  template: 
+    src: templates/gitlab.yml.j2
+    dest: "{{ gitlab_install_path | mandatory }}/config/gitlab.yml"
+
+- name: Copy secrets file
+  template:
```

Comment:
Why is this in the permissions file? Also why don't you set permissions?

Reply:
move some where else